ui/src/testing/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { AppStatus } from 'generated/fetch';
+
+export function minus<T>(a1: T[], a2: T[]): T[] {
+  return a1.filter((e) => !a2.includes(e));
+}
+
+export const ALL_STATUSES = Object.keys(AppStatus)
```

Comment:
we have many kinds of status so I'd prefer ALL_APP_STATUSES or allUserAppStatuses or something like that

Reply:
Renamed to `ALL_GKE_APP_STATUSES` 👍